web/src/compose_banner.ts

```diff
@@ -52,6 +52,8 @@ export const CLASSNAMES = {
     user_not_subscribed: "user_not_subscribed",
 };
 
+export const existing_banners = new Set();
```

Comment:
I think this code will read just a bit more easily if you name this set `visible_banners`.